### ISSUE TITLE
fix rule use-env-prefix

### ISSUE THREAD

**Issue by tyankatsu0105** [gridsome/eslint-plugin-gridsome#116]
**Please write rule name what you want to improve:**

**How are the rules improved?:**
filename is not match pathsForBrowserfileOption

**Provide 2-3 code examples that this rule should warn about:**

```vue

```

**Additional context**
